client/extensions/woocommerce/app/products/product-form-variations-table.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { isNumber } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ProductFormVariationRow from './product-form-variation-row';
+
+class ProductFormVariationsTable extends React.Component {
+
+	static propTypes = {
+		variations: PropTypes.array,
+		product: PropTypes.object,
+		editProductVariation: PropTypes.func.isRequired,
+	};
+
+	renderVariationRow( variation ) {
+		const { product, variations, editProductVariation } = this.props;
+		const id = isNumber( variation.id ) && variation.id || 'index_' + variation.id.index;
+		const manageStock = variations && variations[ 0 ].manage_stock || false;
+
+		return (
+			<ProductFormVariationRow
+				key={ id }
+				product={ product }
+				variation={ variation }
+				manageStock={ manageStock }
+				editProductVariation={ editProductVariation }
+			/>
+		);
+	}
+
+	render() {
+		const { variations, translate } = this.props;
+		return (
+			<div className="products__product-form-variation-table-wrapper">
+				<table className="products__product-form-variation-table">
+					<thead>
+						<tr>
+							<th></th>
+							<th></th>
+							<th className="products__product-price">{ translate( 'Price' ) }</th>
+							<th>{ translate( 'Dimensions & weight' ) }</th>
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'Dimensions' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woocommerce-variation-rows)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).